# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/DMABufSurface.h
# Commit: a8ce70eff417
# Full Hash: a8ce70eff41744ce26905ea160b21b63516c461b
# Author: stransky <stransky@redhat.com>
# Date: 2025-01-08 04:42:02
# Regressor Bug: 1940037
# File Overlap Count: 1
# Description:
#   Bug 1940037 [Wayland] Use exact FOURCC format for DMABuf surfaces r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D233241
# ==============================================================================

diff -r 20a5670ea914 -r a8ce70eff417 widget/gtk/DMABufSurface.h
--- a/widget/gtk/DMABufSurface.h	Tue Jan 07 11:58:50 2025 +0000
+++ b/widget/gtk/DMABufSurface.h	Tue Jan 07 11:58:51 2025 +0000
@@ -219,6 +219,9 @@
   // Surface type (RGBA or YUV)
   SurfaceType mSurfaceType;
 
+  // Actual FOURCC format of whole surface (includes all planes).
+  int32_t mFOURCCFormat = 0;
+
   // Configuration of surface planes, it depends on surface modifiers.
   // RGBA surface may use one RGBA plane or two planes (RGB + A)
   // YUV surfaces use various planes setup (Y + UV planes or Y+U+V planes)
